sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentAction.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * The option bags to use when analyzing sentiment as an action.
+ */
+@Fluent
+public final class AnalyzeSentimentAction {
+    private String actionName;
+    private String modelVersion;
+    private boolean includeStatistics;
```

Comment:
AnalyzeSentimentAction should include all the properties in the AnalyzeSentimentOptions bags, but could have more additional property, ex. the actionName. (If "async API should be able to done the same thing as in sync API", this assumption still valid).

If the Action async API doesn't take include statistics any more, then it can be removed.

Reply:
As I talked to @suhas92, the assumption is still valid.

Reply:
I think I'm still confused, sorry. Do you recommend that users set `includeStatistics` on the individual *Action type or on the `AnalyzeActionOptions`? Is there behavior difference depending on where it's set?

Reply:
Same question as Krista

Reply:
removed the `includeStatistics` from per action option bags since it is either all true or false.